napari/_qt/qt_main_window.py

```diff
@@ -507,7 +507,7 @@ def __init__(self, viewer: 'Viewer', *, show: bool = True):
         viewer.events.help.connect(self._help_changed)
         viewer.events.title.connect(self._title_changed)
         viewer.events.theme.connect(self._update_theme)
-        viewer.layers.events.connect(self.file_menu.update)
+        # viewer.layers.events.connect(self.file_menu.update)
```

Comment:
This was done by Talley, any idea why this was commented out @DragaDoncila or @nclack ?

Reply:
the file menu is no longer a evented model that reacts to these events.